docs/osquery/osquery.asciidoc

```diff
@@ -82,23 +82,24 @@ You can run packs as live queries or schedule packs to run for one or more agent
 
 . Click the **Packs** tab.
 . Click **Add pack** to create a new pack, or click the name of an existing pack, then **Edit** to add queries to an existing pack.
+. Provide a name for the pack. The short description is optional. 
+. Schedule the pack to be deployed on specified agent policies (*Policy*) or on all agent policies (*Global*).
++
+If you choose the *Policy* option, you must also configure one or both of the following fields: 
++
+NOTE: When defining pack deployment details, you cannot configure the same policy multiple times. In other words, after specifying a policy, you can either choose to deploy the pack to all of the policy's agents or only a subset. You cannot choose both.
 
-. Provide the following fields:
-
-* The name of the pack.
-
-* A short description of the pack.
-
-* The agent policies where this pack should run. If no agent policies are set, the pack is not scheduled.
+** *Scheduled {agent} policies (optional)*: Allows you to deploy the pack to specific agent policies. By default, the pack is deployed to all {agents} that are registered to the policies you define.  
+** *Partial deployment (shards)*: Allows you to partially deploy the pack to specified agent policies. After defining a policy, use the *Shard* slider to limit the number of agents to which the pack is deployed. For example, after specifying a policy, you can choose to deploy the pack to half of the policy's agents by selecting 50% on the slider.
 
-. Add queries to schedule:
+. If you are creating a new pack, add queries to schedule:
 
-* To add a query to the pack, click *Add query*, and then either add a saved query or enter a new query.
+ ** Click *Add query* and then add a saved query or enter a new query.
 Each query must include a unique query ID and the interval at which it should run.
 Optionally, set the minimum Osquery version and platform,
 or <<osquery-map-fields,map ECS fields>>. When you add a saved query to a pack, this adds a copy of the query. A connection is not maintained between saved queries and packs.
 
-* To upload queries from a `.conf` query pack, drag the pack to the drop zone under the query table. To explore the community packs that Osquery publishes, click *Example packs*.
+** Upload queries from a `.conf` query pack by dragging the pack to the drop zone under the query table. To explore the community packs that Osquery publishes, click *Example packs*.
```

Comment:
Speaking of config, do you think it might be worth to mention that the policy's agents % is specified in config as 'shard' ?

Reply:
Yes, good reminder! I'll mention that under step 4 in this section and then again under step 5 [here](https://www.elastic.co/guide/en/kibana/current/manage-osquery-integration.html#osquery-custom-config).